### ISSUE TITLE
Is the implementation of reconstructed_probability in the document VAEtf1.py not correct? 

### ISSUE THREAD

**Issue by BuleSky233** [Michedev/VAE_anomaly_detection#13]
Your implementation of reconstructed_probability in the document VAEtf1.py will get the same result in L repetition of sample. I think here the latent parameter z should be sampled L times from the outputs of the decoder, e.g., Normal(\mu_z , \sigma_z).

**Comment by Vhunon** [comment]
Yes, it should draw L samples from the decoder as proposed in http://dm.snu.ac.kr/static/docs/TR/SNUDM-TR-2015-03.pdf Page 9 Algorithm 4 instead of repeating L times.
